Competition2018/src/main/java/competition/subsystems/gripperdeploy/commands/GripperDeployViaJoysticksCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.gripperdeploy.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.gripperdeploy.GripperDeploySubsystem;
+import xbot.common.command.BaseCommand;
+
+public class GripperDeployViaJoysticksCommand extends BaseCommand {
```

Comment:
You also need to set this as the default command for the subsystem (see SubsystemDefaultCommandMap)